pkgs/by-name/br/bruno-cli/package.nix

```
@@ -0,0 +1,77 @@
+{
+  lib,
+  stdenv,
+  buildNpmPackage,
+  bruno,
+  pkg-config,
+  pango,
+  apple-sdk_11,
+}:
+
+buildNpmPackage {
+  pname = "bruno-cli";
+
+  inherit (bruno) version src npmDepsHash;
+
+  npmWorkspace = "packages/bruno-cli";
+  npmFlags = [ "--legacy-peer-deps" ];
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs =
+    [
+      pango
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      # fix for: https://github.com/NixOS/nixpkgs/issues/272156
+      apple-sdk_11
+    ];
```

@mattpolzin can you check if this necessary? Also a quick test on darwin would be welcome :)

```suggestion
    ++ lib.optionals stdenv.hostPlatform.isDarwin [
      # fix for: https://github.com/NixOS/nixpkgs/issues/272156
      apple-sdk_11
    ];
```

---

I can try to find time tonight. A family medical issue has had me away from the computer almost entirely.

---

I can confirm that `apple-idk_11` is indeed required here. It is needed to build a native extension for at least one node dependency. granted that may well be a dependency not even needed for the CLI executable (I'm not sure of it), but that sounds like it's an issue with the Bruno CLI build tooling being under-developed so far.